### ISSUE TITLE
How can i use with code contains async function 

### ISSUE THREAD

**Issue by Suniron** [mysticatea/eslint4b#7]
How to configure this module so that it does not detect an error in the following code:
```async function* main() {/* do something with await*/}```

I got this error message (with `linter.verify())`:
```
{
  ruleId: null, 
  fatal: true, 
  severity: 2, 
  message: 
  "Parsing error: Unexpected token function"
line: 1, column: 7
}
```

I think that i need to set ESLint parserOptions on another ECMAversion but I don't know about that.

Thanks

**Comment by mysticatea** [comment]
> I think that i need to set ESLint parserOptions

Exactly. You can see the official documentation on https://eslint.org/docs/developer-guide/nodejs-api#linter
